src/main/java/org/arl/fjage/connectors/WebServer.java

```diff
@@ -302,4 +309,18 @@ public boolean hasContext(String context) {
     return staticContexts.get(context) != null;
   }
 
+  /**
+   * Adds a rule to rewrite handler.
+   *
+   * @param rule rewrite rule.
+   */
+  public void addRule(Rule rule) {
+    log.info("Adding rewrite rule: "+rule);
```

Comment:
Shall we make this a `log.fine` instead?

Reply:
Modified to `log.fine`.